packages/clerk-js/src/ui/localization/defaultEnglishResource.ts

```diff
@@ -128,6 +134,26 @@ export const defaultResource: DeepRequired<LocalizationResource> = {
       formTitle: 'Verification link',
       formSubtitle: 'Use the verification link sent your email',
       resendButton: 'Resend link',
+      unusedTab: 'You may close this tab',
```

Comment:
❓ Do you think we can nest this one under a key, as the other ones in this list? 
eg

```
unusedTab: {
  title: 'Successfully signed in',
}
```

Im find either way, just a suggestion to make this more consistent and potential more feature proof

Reply:
Yeah, makes sense!